### PR TITLE
Document useful arguments for `wpt run chrome`

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-03-29 09:58:15
Finished: 2018-03-29 10:02:18





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/26624)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/359767291)

**Review by kereliuk — APPROVED** [review]
LGTM, thanks for adding these!

File: docs/_running-tests/chrome.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+```
+./wpt run --binary `which google-chrome-unstable` chrome
+```
+
+Extra arguments to Chrome can be passed with `--binary-args`.
+
+To enable experimental web platform features:
+
+```
+./wpt run --binary-arg=--enable-experimental-web-platform-features chrome fullscreen/
```
Comment (kereliuk) [inline docs/_running-tests/chrome.md]: nit: can you add a link that gives more explanation of what this flag does?

Comment (foolip) [reply docs/_running-tests/chrome.md]: Will link to https://www.chromium.org/blink/runtime-enabled-features and chrome://flags/#enable-experimental-web-platform-features

Comment (foolip) [reply docs/_running-tests/chrome.md]: Well, that was harder than I thought, but PTAL.

File: docs/_running-tests/chrome.md [hunk 2]
```diff
@@ -13,8 +13,7 @@ run Chrome Dev on Linux:
 
 Extra arguments to Chrome can be passed with `--binary-args`.
 
-To enable all [experimental web platform features](https://www.chromium.org/blink/runtime-enabled-features)
-[Experimental Web Platform features](chrome://flags/#enable-experimental-web-platform-features):
+To enable all [experimental web platform features](https://www.chromium.org/blink/runtime-enabled-features) (chrome://flags/#enable-experimental-web-platform-features):
```
Comment (gsnedders) [inline docs/_running-tests/chrome.md]: This seems wrong per the commit message; I presume you meant to keep the https link here.

Comment (foolip) [reply docs/_running-tests/chrome.md]: I did keep one https link, it's just chrome:// which seems impossible to make a link from.

**Pull request merged**